CONTRIBUTING.md

```diff
@@ -109,9 +109,9 @@ An issue that we are not sure we will be doing will not be added to any mileston
 
 A milestone (and hence release) is considered done when all outstanding issues/PRs have been closed or moved to another milestone.
 
-## Semantic Versioning
+## Backwards Compatibility
```

Comment:
I think it's a good idea to call out semantic versioning as our scheme somewhere. This week I came across a project that changed an API in a patch release and it wasn't for security. Not everyone has the same idea of backwards compatibility that we do. It would be good to still have that listed somewhere.

Reply:
good call. I've reverted this change.